responders/src/UI/embc-responder/src/app/app.module.ts

```diff
@@ -28,7 +30,8 @@ import { DatePipe } from '@angular/common';
     }),
     ApiModule.forRoot({ rootUrl: '' }),
     ErrorHandlingModule.forRoot(),
-    SharedModule
+    SharedModule,
+    OutageModule
```

Comment:
If this is shared module, should go to shared module file, if its a feature component its lazy loaded via routing